src/components/mobile/Page.vue

```diff
@@ -140,6 +142,12 @@ export default {
         margin-left: rem(-16px);
         margin-right: rem(-16px);
       }
+
+      .tabs, .date {
```

Comment:
Please, apply these styles locally, for now, I'm not sure how to proceed with them.